packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -101,260 +116,141 @@ export const Default = {
-        }
-      ],
-      args
-    })
+export const Placement = {
+  name: 'Placement',
+  render: () => html`
+    <div class="grid grid-cols-2 gap-32">
+      <sd-dropdown open placement="bottom-start">
+        <div class="slot slot--border slot--text p-4 w-[300px]">Default slot</div>
+        <sd-button slot="trigger">Bottom start</sd-button>
+      </sd-dropdown>
+
+      <sd-dropdown open placement="bottom-end">
+        <div class="slot slot--border slot--text p-4 w-[300px]">Default slot</div>
+        <sd-button slot="trigger">Bottom End</sd-button>
+      </sd-dropdown>
+    </div>
+  `
 };
 
 /**
- * The dropdown automatically flips its position to stay most visible in the viewport. To disable this behavior, set the `no-flip` attribute.
+ * Use the `stay-open-on-select` attribute to keep the dropdown open after an item has been selected.
  */
 
-export const NoFlip = {
-  parameters: { controls: { exclude: ['no-flip', 'default'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'no-flip'
-        }
-      },
-      constants: {
-        type: 'slot',
-        name: 'default',
-        value: '<div class="example"></div>'
-      },
-      args
-    })
+export const StayOpen = {
+  name: 'Stay Open on Select',
+  render: () => html`
+    <sd-dropdown stay-open-on-select>
+      <div class="example">
+        <sd-checkbox>Checkbox 1</sd-checkbox>
+        <sd-checkbox>Checkbox 2</sd-checkbox>
+        <sd-checkbox>Checkbox 3</sd-checkbox>
+      </div>
+      <sd-button slot="trigger">Trigger</sd-button>
+    </sd-dropdown>
+  `
 };
 
 /**
- * The preferred placement of the dropdown can be set with the `placement` attribute.
- * Note that the actual position may vary to ensure the panel remains in the viewport.
+ * Use the `disabled` attribute to disable the dropdown.
  */
 
-export const Placement = {
-  parameters: { controls: { exclude: ['placement'] } },
-  render: (args: any) => {
-    return html`<div class="placement-story">
-      ${['top', 'bottom', 'left', 'right'].map(value =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'attribute',
-              name: 'placement',
-              values: [value, `${value}-start`, `${value}-end`]
-            }
-          },
-          args,
-          constants: [
-            {
-              type: 'template',
-              name: 'placement',
-              value: `<div class="template-placement">%TEMPLATE%</div>`
-            },
-            {
-              type: 'attribute',
-              name: 'no-flip',
-              value: true
-            }
-          ]
-        })
-      )}
-    </div> `;
-  }
+export const Disabled = {
+  name: 'Disabled',
+  render: () => html`
+    <sd-dropdown disabled>
+      <div class="slot slot--border slot--text p-4 w-[300px]">Default Slot</div>
+      <sd-button slot="trigger">Trigger</sd-button>
+    </sd-dropdown>
+  `
```

Comment:
I'm not sure, if this is right. Disabled shouldn't have a primary background because right now it looks like you can click on it.
![image](https://github.com/user-attachments/assets/35c4b812-1cd8-4b7b-9eff-7fa8b4e10546)

Reply:
I agree that it looks strange, but to change the trigger background color we also need to either target it with css or use the disabled attribute in the slotted component. I've updated the example by adding the attribute to the sd-button trigger but I feel this should also be documented next to the attribute description. What do you think @yoezlem?